spec/models/ccms/submission_spec.rb

```diff
@@ -15,6 +15,38 @@ module CCMS
              case_poll_count:
     end
 
+    describe "after_commit callback" do
```

Comment:
nit, not a blocker. I prefer to use the method definition syntax `.` for class method `#` for instance method
```suggestion
    describe ".after_commit" do
```